dnanexus/src/eggd_athena.sh

```
@@ -0,0 +1,86 @@
+    [ -n "$panel_filters" ] && panel_filters="--panel_filters ${panel_filters} " || unset panel_filters
+    [ -n "$indication" ] && indication="--indication '${indication}' " || unset indication
+}
+
+_upload_outputs() {
+    report=$(find . -type f -maxdepth 1 -name "*_coverage_report.html")
+    gene_coverage=$(find . -type f -maxdepth 1 -name "*.gene_coverage.tsv")
+    region_coverage=$(find . -type f -maxdepth 1 -name "*.region_coverage.tsv")
+    annotated_bed=$(find . -type f -maxdepth 1 -name "*.coverage.bed")
+    summary_text=$(find . -type f -maxdepth 1 -name "*_summary.txt")
+
+    dx-jobutil-add-output report $(dx upload "$report" --brief) --class=file
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)